### PR TITLE
[#657] requeue on resource error, tidy requeue logic, allow verbose s…

### PR DESCRIPTION
**Pull request by gtully**
…etting for tests via env

### CODE REVIEW

File: controllers/activemqartemis_controller.go [hunk 1]
```diff
@@ -137,61 +137,56 @@ func (r *ActiveMQArtemisReconciler) Reconcile(ctx context.Context, request ctrl.
...
+	if requeueRequest {
+		reqLogger.V(1).Info("requeue reconcile")
+		result = ctrl.Result{RequeueAfter: common.GetReconcileResyncPeriod()}
+	}
+
 	return result, err
 }
 
-func validate(customResource *brokerv1beta1.ActiveMQArtemis, client rtclient.Client, scheme *runtime.Scheme, namer Namers) (bool, ctrl.Result) {
+func validate(customResource *brokerv1beta1.ActiveMQArtemis, client rtclient.Client, scheme *runtime.Scheme, namer Namers) (valid bool, retry bool) {
```
Comment (brusdev) [inline controllers/activemqartemis_controller.go:189]: Where are you using the named return arguments?

File: controllers/activemqartemis_controller.go [hunk 2]
```diff
@@ -137,61 +137,56 @@ func (r *ActiveMQArtemisReconciler) Reconcile(ctx context.Context, request ctrl.
...
-
-	if err != nil {
-		if apierrors.IsConflict(err) {
-			reqLogger.V(1).Info("unable to update ActiveMQArtemis status", "Request Namespace", request.Namespace, "Request Name", request.Name, "error", err)
-			err = nil // we don't want the controller event loop reporting this as an error
-		} else {
-			reqLogger.V(1).Error(err, "unable to update ActiveMQArtemis status", "Request Namespace", request.Namespace, "Request Name", request.Name)
-		}
-		return ctrl.Result{RequeueAfter: common.GetReconcileResyncPeriod()}, err
+	crStatusUpdateErr := UpdateCRStatus(customResource, r.Client, request.NamespacedName)
```
Comment (brusdev) [inline controllers/activemqartemis_controller.go:168]: Did you remove the dump of the err from the log on purpose?

**Review by brusdev — APPROVED** [review]

File: controllers/activemqartemis_controller.go (hunk 2 above)
Comment (gtully) [reply controllers/activemqartemis_controller.go:168]: yes, it was a duplicate, already logged by resources.UpdateStatus, good watching!

File: controllers/activemqartemis_controller.go (hunk 1 above)
Comment (gtully) [reply controllers/activemqartemis_controller.go:189]: I was only using one, the retry. Can remove the name from the first bool, will do that. thanks

Comment (gtully) [reply controllers/activemqartemis_controller.go:189]: I did not use a naked return, but I think it helps to use the named argument in this case.

**Comment by gtully** [comment]
@brusdev thanks for the fast feedback :-)

**Pull request merged**